### PR TITLE
Merge: Merge Minim Labs and 3rd parties integration open source into one

### PR DESCRIPTION
**Pull request by venkatchimata**

### CODE REVIEW

**Review by tyler-sommer — DISMISSED** [review]

**Review by adamsmola — DISMISSED** [review]
LGTM

File: README-lede_generic.md [hunk 1]
```diff
@@ -15,3 +15,32 @@ The agent has its own version that is generated if it is built separately
 from the firmware (only happens during development, it is based on time).
 For the release builds the agent version and the firmware version are the
 same.
+
+## Platforms
+
+Following platforms are supported so far.
+ - TPLink Archer C7 v2/v4/v5
+ - GL - Inet B1300
+ - GL - Inet AR300M
```
Comment (dob71) [inline README-lede_generic.md]: We aren't building AR300M and Linksys.
You can list them as "support being phased out" along with Smatrg SR400 (there are still files for all 3).

File: README-lede_generic.md [hunk 2]
```diff
@@ -15,3 +15,32 @@ The agent has its own version that is generated if it is built separately
...
+
+## Build
+
+ LEDE code with unum-specifc build config files is here:
+ https://github.com/violetatrium/lede
+
+ Make sure unum-sdk is accessisble from lede as ../unum-sdk.
+
+ Build can be triggered with the following command from lede directory:
+ ./build -d -e <platform name>
```
Comment (dob71) [inline README-lede_generic.md]: No need to build with '-e' anymore, the build script now builds with multiple CPUs then switches to -j1 if error is detected.

File: dist/README-dist.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Distribution-specific builds
+
+Maintainer notes for building and publishing various "finished product"
+produced by or with the Unum SDK.
+
+- [OpenWrt (.ipk and full images)][2]
```
Comment (dob71) [inline dist/README-dist.md]: If we are dropping openwrt_generic, so no more .ipk
and the references to the openwrt builds below will have to go

Comment (venkatchimata) [reply dist/README-dist.md]: Agreed. Removed openwrt_generic references from the documentation.

File: README-lede_generic.md (hunk 2 above)
Comment (venkatchimata) [reply README-lede_generic.md]: Removed option '-e'.

File: README-lede_generic.md (hunk 1 above)
Comment (venkatchimata) [reply README-lede_generic.md]: OK. Done.

**Review by dob71 — APPROVED** [review]
Please squash the commits when merging

**Review by tyler-sommer — APPROVED** [review]
The reorganized readme looks great!

**Review by adamsmola — APPROVED** [review]
:+1:

**Pull request merged**